### ISSUE TITLE
Cluster with multiple instances in SVL 2021.1?

### ISSUE THREAD

**Issue by andreas-kustermann** [lgsvl/simulator#1423]
@EricBoiseLGSVL 
When setting up a distributed local simulation in SVL 2021.1 is there a way to add more than 1 simulation instance in client mode? 
1 master and 1 client only works fine.
`"Something went wrong. Cluster was not created."` is the error message by the UI when trying to add any 2nd simulation instance in client mode to our existing cluster. 
We tried this with different machines in our local network, all were compatibel as first client, none as second. Using the default linux build on all machines without changes so far.

Thank you very much for your help!
Kind regards, Andreas

**Comment by EricBoiseLGSVL** [comment]
Yes but you have a limited amount of clusters in webUI.  @zelenkovsky

**Comment by EricBoiseLGSVL** [comment]
@andreas-kustermann yes we have a limit of 2 instances, sorry for the confusion.

**Comment by zelenkovsky** [comment]
@andreas-kustermann for all users, by default, we have a limit of 2 instances per cluster. I could lift the limit - please drop a message to Dmitry.Zelenkovsky(at)lge.com and provide your account's email address. I will help.

**Comment by andreas-kustermann** [comment]
Thank you guys for the assistance, highly appreciated, will send you an email asap. Best, Andreas
